front-end/src/components/Admin/Record/RecordEditor/index.js

```diff
@@ -100,24 +93,18 @@ class RecordEditor extends Component {
 
     return (
       <Fragment>
-        {this.state.loading ? <div className="record__loader-wrapper">
-          <FadeLoader
-            sizeUnit={'px'}
-            size={50}
-            color={'#999'}
-            loading={this.state.loading}
-          />
-        </div> : <Form
+        <Form
           schema={entity.schema}
           uiSchema={entity.uiSchema}
           formData={getRecordData()}
           autocomplete='off'
           onChange={this.changeData}
           onSubmit={this.saveData}
-          onError={this.log('errors')}
-        >
-          <button className='record__button'>Сохранить</button>
-        </Form>}
+          onError={this.log('errors')}>
+          <button ref={saveButton => (this.saveButton = saveButton)} onClick={this.saveData} className='record__button'>Сохранить</button>
```

Comment:
Here is an example of how you control an element state (enabled/disabled). It has to be based on component state. We don't need refs here.
```
<button disabled={this.state.loading} onClick={this.saveData} className='record__button'>Сохранить</button>
```

Reply:
that's right )